flask_dance/contrib/salesforce.py

```diff
@@ -39,6 +40,9 @@ def make_salesforce_blueprint(
         reprompt_consent (bool): If True, force Salesforce to re-prompt the user
             for their consent, even if the user has already given their
             consent. Defaults to False.
+        domain (str, optional): The domain of your Salesforce instance. By default,
```

Comment:
Does the Salesforce documentation refer to this as a "domain" or a "subdomain"? Technically, this is a subdomain of the `salesforce.com` domain, but we should probably follow whatever terminology is in the Salesforce documentation. I have never used the Salesforce API, so I leave this up to you.

Reply:
Yes this is a subdomain but leaded me to that decision was that Salesforce uses `My Domain` naming for possibility to customize instance's URL (https://help.salesforce.com/articleView?id=sf.domain_name_overview.htm&type=5). It's not clear so if you think the subdomain will be more clear to the user (domain can suggest inserting full domain name) I can change it. But keep in mind that user needs to put sometimes more subdomains like (example.my) so subdomain naming can also be misleading.

Reply:
It solves the issue of domain vs subdomain: https://github.com/singingwolfboy/flask-dance/pull/346/commits/db8ad30fefcb61bb6cc7370d30a98d9ac3b807cf